youtube_dl/extractor/wimp.py

```diff
@@ -45,13 +51,23 @@ def _real_extract(self, url):
                 'ie_key': YoutubeIE.ie_key(),
             }
 
-        info_dict = self._extract_jwplayer_data(
-            webpage, video_id, require_title=False)
+        url = self._search_regex(r'class=\'oyt-container .*? data-src=\'(.*?)\'', webpage, 'Video URL', default=None, fatal=False)
```

Comment:
Do not shadow `url` variable.  `fatal` has no effect when `default` is provided.